generateSchema.js

```diff
@@ -437,14 +439,25 @@ function processModels() {
         return model && model.typeId;
     }
     function isSchemable(modelName) {
-        return modelName.match(/Catalog(Item|Group|Member)\.js$/) &&                   
+        return modelName.match(/Catalog(Item|Group|Member)\.js$/) &&
               !modelName.match(/(ArcGisMapServerCatalogGroup|addUserCatalogMember)/);
```

Comment:
I don't really remember why ArcGisMapServerCatalogGroup is a special case, but perhaps that should be generalised too?